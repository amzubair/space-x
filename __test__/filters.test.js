import React from "react";
import renderer from "react-test-renderer";
import { render } from "@testing-library/react";
import Filters from "../component/filters";

const router = {
  route: "/",
  pathname: "/",
  query: { launch_year: "2019" },
  asPath: "/?launch_year=2019",
  basePath: "",
  events: undefined,
  isFallback: false,
};

test("renders developed by", () => {
  const tree = renderer.create(<Filters router={router} />).toJSON();
  expect(tree).toMatchSnapshot();
});
