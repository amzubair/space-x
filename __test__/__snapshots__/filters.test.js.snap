// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders developed by 1`] = `
<div
  className="filters"
>
  <div
    className="filters-container card"
  >
    <div
      className="filters-section "
    >
      <button
        className="filters-button filters-clear"
        onClick={[Function]}
      >
        Clear Filters
      </button>
      <h2>
        Year
      </h2>
      <button
        className="filters-button "
        onClick={[Function]}
      >
        2006
      </button>
      <button
        className="filters-button "
        onClick={[Function]}
      >
        2007
      </button>
      <button
        className="filters-button "
        onClick={[Function]}
      >
        2008
      </button>
      <button
        className="filters-button "
        onClick={[Function]}
      >
        2009
      </button>
      <button
        className="filters-button "
        onClick={[Function]}
      >
        2010
      </button>
      <button
        className="filters-button "
        onClick={[Function]}
      >
        2011
      </button>
      <button
        className="filters-button "
        onClick={[Function]}
      >
        2012
      </button>
      <button
        className="filters-button "
        onClick={[Function]}
      >
        2013
      </button>
      <button
        className="filters-button "
        onClick={[Function]}
      >
        2014
      </button>
      <button
        className="filters-button "
        onClick={[Function]}
      >
        2015
      </button>
      <button
        className="filters-button "
        onClick={[Function]}
      >
        2016
      </button>
      <button
        className="filters-button "
        onClick={[Function]}
      >
        2017
      </button>
      <button
        className="filters-button "
        onClick={[Function]}
      >
        2018
      </button>
      <button
        className="filters-button selected"
        onClick={[Function]}
      >
        2019
      </button>
      <button
        className="filters-button "
        onClick={[Function]}
      >
        2020
      </button>
    </div>
    <div
      className="filters-section"
    >
      <h2>
        Successful Launch
      </h2>
      <button
        className="filters-button "
        onClick={[Function]}
      >
        True
      </button>
      <button
        className="filters-button "
        onClick={[Function]}
      >
        False
      </button>
    </div>
    <div
      className="filters-section"
    >
      <h2>
        Successful Landing
      </h2>
      <button
        className="filters-button "
        onClick={[Function]}
      >
        True
      </button>
      <button
        className="filters-button "
        onClick={[Function]}
      >
        False
      </button>
    </div>
  </div>
</div>
`;
